src/client.rs

```diff
@@ -204,7 +204,7 @@ pub async fn client_entrypoint(
                 Ok(mut client) => {
                     info!("Client {:?} connected (plain)", addr);
 
-                    if client.is_admin() {
+                    if !client.is_admin() {
```

Comment:
This is the opposite of what we're doing now. Could you describe the bug you're seeing?

Reply:
Recreating the bug.

Start pgcat.

Create client with sslmode disabled, `?sslmode=disable`
execute single query then close connection.

Send sigint to pgcat

Reply:
this is actually most likely a typo since in every other case we're doing `!client.is_admin()`

Reply:
You're right, very nice find.